### ISSUE TITLE
User Mailing List

### ISSUE THREAD

**Issue by csenol** [p2t2/figaro#582]
Hi

I started reading the book and playing with figaro. I think, it would be useful if you'd have a user mailing list for discussions and questions

**Comment by apfeffer** [comment]
Good idea. We'll look into this. In the meantime, you can use the manning book forum.
